Makefile

```diff
@@ -0,0 +1,5 @@
+test:
+	PYTHONPATH=. pytest tests
+
+test-coverage:
+	PYTHONPATH=. pytest --cov=apistar --cov-report xml --cov-report term tests
```

Comment:
Let's use GitHub scripts style instead, adding `/scripts/test`, and `/scripts/test --cov`.